workspace.jsonc

```diff
@@ -8,7 +8,7 @@
     // "vendorDirectory": "vendor",
   },
   "teambit.dependencies/dependency-resolver": {
-    "packageManager": "teambit.dependencies/yarn",
+    "packageManager": "teambit.dependencies/pnpm",
```

Comment:
don't commit this or Gilad will get upset

Reply:
I change it back.